src/client/Microsoft.Identity.Client/Internal/Logger/MsalLegacyIdentityLogger.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.IdentityModel.Abstractions;
+
+namespace Microsoft.Identity.Client.Internal.Logger
+{
+    internal class MsalLegacyIdentityLogger : IIdentityLogger
```

Comment:
I guess that since we're not deprecating the callback approach, a better name would be "CallbackIdentityLogger".